src/orchestrator.ts

```diff
@@ -167,11 +159,14 @@ export class Orchestrator {
                     return;
                 }
 
-                const newDecisionStartedEvent = state.find(
-                    (e) =>
-                        e.EventType === HistoryEventType.OrchestratorStarted &&
-                        e.Timestamp > decisionStartedEvent.Timestamp
-                );
+                const newDecisionStartedEvent = state

```

Comment:
Can we explicitly add a comment that we grab the last `OrchestratorStarted` event before our current place in the history?

Reply:
Added!